### DISCUSSION TITLE
How can I integrate the idea of zettelkasten or making evergreen notes with taking class notes?

### DISCUSSION THREAD

**Discussion by linggaaradhanas22** [BryanJenksCommunity/FAQ#24] [Q&A] [answered]
Hello Bryan!
Before I began to ask my question, let me give you some backgrounds. I am an undergrad freshman, currently  studying electrical engineering (wish me luck 🙂). And for the past few months, I’ve been digging myself deep into the topics of PKM and stuff like that, but more spesifically the workflow itself.

Now, going back to the question, 
How can I integrate the idea of zettelkasten or making evergreen notes with taking class notes? The thing is, I love the idea of interconnectedness of knowledge and I want to apply that philosophies to what I’m studying, not just for when I consume content on the internet.

If that’s to broad to answer, here’s a more spesific question:
Do you consider class notes as a literature notes and then from there you distill certain ideas and then turn them into evergreen notes, or just consider them a totally different thing?

I want to create my review notes based on my evergreen notes that I’ve took so that it totally reflects my understanding of the subject which in the end will help me developed a mental model for that subject.

Anyway sorry for the long message 😉

**Comment by tallguyjenks** [comment, marked as answer]
When it comes to Zettelkasten and Notes from school course work I treat them as 2 completely different realms of information.

To me, a Zettelkasten is a collection of ideas, concepts, and thoughts, linked together for the purpose of creative output. 
Academic note-taking is to aid in the retention and assimilation of course material to pass a test and/or gain understanding on a subject that you would like to develop mastery in. I myself use Obsidian for both Zettelkasten Notes and purely academic notes for my university course work.

How I separate them comes down to maps of content. see #14 #18 for MOC's. I have a MOC node for my school [[WGU]] and in that note, I link to my courses [[C175 Database Management]], etc. and from there I may link to a bullet outline for topics/chapters/subjects/ desired learning goals/course objectives, etc. and I will take notes that link back to their parent topics/sections/etc. this way the academic notes are cordoned off from the Zettelkasten notes. They live in the same vault and are searchable and linked through the MOC's, but they are not my own unique ideas, and sure they may be factual material, but not all of them may be relevant to my growing body of notes. but I still use those notes for school.

For academic study, I also like to utilize Anki for spaced repetition of material and I may also use Obsidian with the Anki plugin to generate flashcards from notes, but ultimately I treat my academic notes and my Zettelkasten notes are 2 separate entities.

**Reply by kritika-gupta** [reply]
Hi Bryan, I was struggling with the same question and your reply makes a lot of sense! Going a bit deeper into "class notes", rather, any factual or technical notes - how do you manage notes taken about the same topic from different sources? For example, if you watch a course video on Linear Regression and take notes in Note A, then read a separate article on Linear Regression (perhaps this has a different way of explaining what the previous source covered, or some additional sub-topics) and take notes in Note B, then would you just link both notes in an MOC about Linear Regression? Or would you come back to your notes and create a different Note C that combines the information you learned via both the sources? 
I generally read about a topic from multiple sources and have been trying to figure out a good way to capture this workflow in Obsidian in an efficient manner!

**Reply by tallguyjenks** [reply]
Linear regression. (LR)

I have a statistics MOC, if i read 2 articles on LR then each article has their "literature" notes pages following my normal convention `( YYYY-MM-DD Article Name` any relevant notes are then added to the actual LR note. or I might simply add the information to the LR note and in the footer portion of that note under `References:` put the link to the source material. It really depends but I use both approaches :)
